src/game/Person.java

```diff
@@ -41,8 +56,17 @@ public void receiveDamage(int damage)
     }
 
 
-
-    public int getRoomId() {return roomId; }
+    public void setRoomId(int roomId) {
+        if(roomId >= boardedShip.getRoomCount()) {
+            throw new RuntimeException("There is no room with this id");
+        }
+        this.roomId = roomId;
+    }
+    public int getRoomId() {
+        if(isMoving)
```

Comment:
Teoretycznie to zależność konkretnej konwencji w pisaniu, ale lepiej zrobić:
`if (isMoving) {
  return 0;
}`
To, że java pozwala dawać jednolinijkowe scopy bez klamer, nie oznacza, że dobrze jest z tego korzystać (z drobnymi wyjątkami).
Co do samej konwencji, którą stosuje 99% firm: https://www.geeksforgeeks.org/java-naming-conventions/